contracts/test/umbra-test.js

```diff
@@ -391,7 +391,7 @@ describe('Umbra', () => {
   it('should revert on meta withdrawal if the fee is more than the amount', async () => {
     const bigFee = sumTokenAmounts([metaTokenTotal, '100']);
 
-    const { v, r, s } = await signMetaWithdrawal(metaWallet, ctx.chainId, ctx.version, metaAcceptor, ctx.token.address, relayer, relayerTokenFee);
+    const { v, r, s } = await signMetaWithdrawal(metaWallet, ctx.chainId, ctx.version, metaAcceptor, ctx.token.address, relayer, bigFee);
```

Comment:
Why was this changed from `relayerTokenFee` to `bigFee`?

Reply:
Good question! Apparently, this test was reverting with the right error but for the wrong reason. It *should* have been using `bigFee` all along. Refactoring the contract internals exposed the incorrect test, so I had to fix it.